client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -31,7 +34,41 @@ const Header: FunctionComponent< Props > = ( {
 	toggleGeneralSidebar,
 	toggleSidebarShortcut,
 } ) => {
-	const { siteTitle, siteVertical } = useSelect( select => select( STORE_KEY ).getState() );
+	const [ domainText, setDomainText ] = useState< string | null >( null );
```

Comment:
Do we need the `| null`? Can we use empty string instead? (To keep the type more... homogenous?)

Reply:
```suggestion
	const [ domainText, setDomainText ] = useState< string >( '' );
```

Reply:
Actually :sweat_smile: 

```suggestion
	const [ domainText, setDomainText ] = useState< DomainName >( '' );
```